kura/org.eclipse.kura.web2/OSGI-INF/metatype/org.eclipse.kura.web.Console.xml

```diff
@@ -43,7 +43,7 @@
             cardinality="0"
             required="true"
             default="/admin/console"
-            description="Default Web Server Entry Point for Applications running of framework\'s Web Server">
+            description="Default Web Server Entry Point for Applications running on the framework's Web Server">
```

Comment:
hey @MMaiero, I have checked this PR and it fixes the issue.
Please update the header and then it can be merged.